pkg/scheduler/apis/config/v1alpha1/conversion_test.go

```diff
@@ -191,3 +193,206 @@ func TestConfigToV1alpha1KubeSchedulerLeaderElectionConfiguration(t *testing.T)
 		}
 	}
 }
+
+func TestConvertBetweenV1Alpha1PluginsAndConfigPlugins(t *testing.T) {
+	// weight is assigned to score plugins
+	weight := int32(10)
+	// DummyWeight is a palceholder for the v1alpha1.plugins' weight will be filled with zero when
+	// convert back from config.
+	dummyWeight := int32(0)
+	v1alpha1Plugins := v1alpha1.Plugins{
+		QueueSort: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "queuesort-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-queuesort-plugin", Weight: &dummyWeight},
+			},
+		},
+		PreFilter: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "prefilter-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-prefilter-plugin", Weight: &dummyWeight},
+			},
+		},
+		Filter: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "filter-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-filter-plugin", Weight: &dummyWeight},
+			},
+		},
+		PostFilter: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "postfilter-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-postfilter-plugin", Weight: &dummyWeight},
+			},
+		},
+		Score: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "score-plugin", Weight: &weight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-score-plugin", Weight: &weight},
+			},
+		},
+		Reserve: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "reserve-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-reserve-plugin", Weight: &dummyWeight},
+			},
+		},
+		Permit: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "permit-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-permit-plugin", Weight: &dummyWeight},
+			},
+		},
+		PreBind: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "prebind-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-prebind-plugin", Weight: &dummyWeight},
+			},
+		},
+		Bind: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "bind-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-bind-plugin", Weight: &dummyWeight},
+			},
+		},
+		PostBind: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "postbind-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-postbind-plugin", Weight: &dummyWeight},
+			},
+		},
+		Unreserve: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "unreserve-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-unreserve-plugin", Weight: &dummyWeight},
+			},
+		},
+	}
+	configPlugins := config.Plugins{
```

Comment:
`wantConfigPlugins` or just `wantConfig`

Reply:
sure

Reply:
Actually, I found it's not a pure `wantConfigPlugins`. This value is used for double check here. It works like `wantConfigPlugins` when compared with a converted v1alpha1 config. And it works as an input config when converting `to` v1alpha1 config. So I think it's better to keep this name.